specs/perfmon/performance-monitoring-vocab.ttl

```diff
@@ -1,4 +1,4 @@
-@prefix qudt:  <http://qudt.org/1.1/schema/qudt> .
+@prefix qudt:  <http://qudt.org/1.1/schema/qudt#> .
```

Comment:
Good catch!

Reply:
The Eclipse plugin XTurtle caught it - it checks that all the vocabulary terms used are actually defined, looking in a cache of common vocabs it keeps. Sounds good - unfortunately the implementation is full of bugs, so I'd not recommend it as a common plugin for everyone.